src/app/ubs/ubs-admin/components/ubs-admin-address-details/ubs-admin-address-details.component.html

```diff
@@ -13,7 +13,7 @@ <h1 class="form__headline py-3">{{ 'address-details.title' | translate }}</h1>
             <select
               [formControlName]="getLangValue('addressRegion', 'addressRegionEng')"
               (change)="setRegionValue($event)"
-              class="shadow-none form-control placeholder-positioning"
+              [ngClass]="{ 'shadow-none': true, 'form-control': true, 'placeholder-positioning': true, 'disables-input': isStatus }"
```

Comment:
Maybe it will be better to use [disabled] attribute to disable input in some condition?